### PR TITLE
[SwiftSyntax] Add more types in protocol map

### PR DESCRIPTION
**Pull request by kimdv**
https://github.com/apple/swift-syntax/pull/331

### CODE REVIEW

**Comment by kimdv** [comment]
https://github.com/apple/swift-syntax/pull/331

@swift-ci Please smoke test

**Comment by kimdv** [comment]
https://github.com/apple/swift-syntax/pull/331

@swift-ci Please smoke test macOS

**Comment by kimdv** [comment]
https://github.com/apple/swift-syntax/pull/331

@swift-ci Please smoke test

**Review by ahoppen — APPROVED** [review]

File: utils/gyb_syntax_support/protocolsMap.py [hunk 1]
```diff
@@ -14,6 +20,15 @@
     'MemberDeclList': [
         'MemberDeclBlock'
     ],
+    'FunctionCallExpr': [
+        'CodeBlockItem',
```
Comment (ahoppen) [inline utils/gyb_syntax_support/protocolsMap.py:24]: For a future PR: Would it make sense to add automatically conformances like `ExpressibleAsFunctionCallExpr : ExpressibleAsExpr` (just like we have `FunctionCallExpr : ExprBuildable` and then add `ExpressibleAsExpr : ExpressibleAsCodeBlockItem`? That way, we wouldn’t need to add conformances to `ExpressibleAsCodeBlockItem` for every expression node.

The same thing I just showed for expression also applies to statements.

Comment (kimdv) [reply utils/gyb_syntax_support/protocolsMap.py:24]: Good idea!

**Pull request merged**